charts/rasa-x/templates/rasa-config-files-configmap.yaml

```diff
@@ -45,7 +45,7 @@ data:
       {{- end }}
       {{- end }}
     action_endpoint:
-      url: "http://{{ include "rasa-x.fullname" . }}-app:{{ template "rasa-x.custom-actions.port" . }}{{ .Values.app.endpoints.actionEndpointUrl }}"
+      url: "{{  .Values.app.scheme }}://{{ include "rasa-x.fullname" . }}-app.{{ .Release.Namespace }}.svc:{{ template "rasa-x.custom-actions.port" . }}{{ .Values.app.endpoints.actionEndpointUrl }}"
```

Comment:
we should also add this to the values.yaml for app, right?

Reply:
Yes! Good 👀  @erohmensing 😄